lib/Util.ts

```diff
@@ -98,6 +98,19 @@ export async function decompressGzippedContent(buffer: Buffer, charset?: string)
     })
 }
 
+/**
+ * Builds a RegExp to test urls against for deciding
+ * wether to bypass proxy from an entry of the
+ * environment variable setting NO_PROXY
+ *
+ * @param {string} bypass
+ * @return {RegExp}
+ */
+export function buildProxyBypassRegexFromEnv(bypass : string) : RegExp {
+    let wildcardEscaped = bypass.replace('*', '(.*)');
```

Comment:
I'm hesitant to do this simply because this behavior has been around for years, and we risk breaking people who have already done something to work around this. I generally think its the right behavior though - I might prefer something like:

```
try {
   // We need to keep this around for back-compat purposes
   return new RegExp(bypass, 'i');
catch (<Exception type (I don't know offhand what this is)>) {
   let wildcardEscaped = bypass.replace('*', '(.*)');
   return new RegExp(wildcardEscaped, 'i');
}

Reply:
I understand what you mean. I changed my implementation to escape the wildcard at the start only if it causes this problem, as suggested.
Unfourtunately it seems that the `https://httpbin.org/redirect-to` endpoint is broken, which causes integration tests to fail.

Reply:
If you pull master into your branch it should be fixed (as of 30 seconds ago 😄)

Reply:
done 👍